app/views/spaceships/index.html.erb

```diff
@@ -1,35 +1,37 @@
 <div class="wrapper">
-  <header>
-    <div class="div--logo">
-      <a href="#"><img src="" alt=""></a>
+  <div>
+    <div class="main--header">
+      <div class="div--logo">
+        <a href="#"><i class="fab fa-android logo"></i></a>
+      </div>
+      <div class="menu">
+        <a href="">Stays</a>
+        <a href="">Host a home</a>
+        <a href="">Help</a>
+        <a href="">Sign Up</a>
+        <a href="">Log In</a>
+      </div>
     </div>
-    <div>
-      <a href="">Stays</a>
-      <a href="">Host a home</a>
-      <a href="">Help</a>
-      <a href="">Sign Up</a>
-      <a href="">Log In</a>
-    </div>
-  </header>
-
+  </div>
 </div>
 
+<div class="my--container">
+  <div class="all-space--container">
```

Comment:
@postolachivlad what means "all-space--container"? (also sometimes 1 and 2 dashes)